INSTALL.rst

```diff
@@ -34,7 +113,8 @@ We build assets using `npm`. Make sure you have installed it system wide.
 
 .. code-block:: console
 
-    (inspirehep)$ npm update && npm install -g node-sass clean-css requirejs uglify-js
+    (inspirehep)$ sudo npm update
+    (inspirehep)$ sudo npm install -g node-sass clean-css requirejs uglify-js
```

Comment:
Found them :smile: (thanks @jacquerie), though I don't think it should be added to this pull request right now (it's not just executing a command, but you have to pull in the invenio scripts and setup env vars and such). I think that should go on it's own pr, once we (aka me) have a clear idea of the setup process.